src/App.tsx

```diff
@@ -14,11 +14,60 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selected: string;
+};
+
+class App extends Component<{}, State> {
+  state = {
+    selected: 'Jam',
+  };
+
+  handleSelect = (good: string) => {
+    this.setState({ selected: good });
+  };
+
+  render() {
+    const { selected } = this.state;
+
+    return (
+      <div className="App">
+        <button
+          type="button"
+          className={selected ? '' : 'App__hiden'}
```

Comment:
```suggestion
          className={selected && 'App__hiden'}
```
you can use 'classnames' library for such cases